projects/driver_controls/src/main.c

```diff
@@ -182,16 +185,18 @@ int main(void) {
         case INPUT_EVENT_DRIVE_UPDATE_REQUESTED:
         case INPUT_EVENT_CAN_RX:
         case INPUT_EVENT_CAN_TX:
+        case INPUT_EVENT_MECHANICAL_BRAKE_PRESSED:
+        case INPUT_EVENT_MECHANICAL_BRAKE_RELEASED:
+        case INPUT_EVENT_SPEED_UPDATE:
+        case INPUT_EVENT_PEDAL_FAULT:
           break;
         default:
           LOG_DEBUG("e %d %d\n", e.id, e.data);
       }
 #endif
       can_process_event(&e);
       power_distribution_controller_retry(&e);
-      cruise_handle_event(cruise_global(), &e);
       event_arbiter_process_event(&s_event_arbiter, &e);
-      brake_signal_process_event(&e);
```

Comment:
If we aren't processing brake signal events anymore in the event loop (via `brake_signal_process_event`), where are we processing them? How are we signalling brake lights? If we aren't signalling lights via this, do we still need the brake signal abstraction now?